src/readit/details/comments/index.js

```diff
@@ -311,11 +341,9 @@ function Details(
                                         </select>
                                     </div>
                                     <hr/>
-                                </div>
-                                }
+                                </div>}
                                 <div className="row">
                                     <ul className="list-group">
-
                                         {comments.map(comment => <CommentItem comment = {comment} canEdit={true}/>)}
```

Comment:
the canEdit should be changed to true only if the currentUser's role is critic or author... not sure if this is the right place to put that check